### DISCUSSION TITLE
Regarding First Ledger Bot of XRP in telegram, my withdraw activity was failed because of no destination tag not address.

### DISCUSSION THREAD

**Discussion by jacobkoo153** [GitbookIO/community#587] [1. How to]
Regarding First Ledger Bot of XRP in telegram, my withdraw activity was failed because of no destination tag not address.
There is none of query of "destinatino tag". How can i do? Please help me.

**Comment by aikoii** [comment]
Hey @jacobkoo153, it seems your question relates to content that may be hosted on GitBook by another user, which we don't manage. Please contact the content owner directly for assistance, or let me know if I'm missing anything.
